strongbox.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Type=Application
+Name=Strongbox
+Exec=AppRun %F
```

Comment:
what does %F do?

Reply:
> A list of files. Use for apps that can open several local files at once. Each file is passed as a separate argument to the executable program.

and

> A command line may contain at most one %f, %u, %F or %U field code. If the application should not open any file the %f, %u, %F and %U field codes must be removed from the command line and ignored.

from 

https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s07.html

removed.